packages/webapp/components/layouts/AccountLayout/AccountContentSection.tsx

```diff
@@ -8,13 +8,13 @@ interface ClassName {
 
 interface AccountContentSectionProps {
   title: string;
-  description?: string;
+  description?: ReactNode;
   children?: ReactNode;
   className?: ClassName;
 }
 
-const ContentHeading = classed('h2', 'font-bold typo-body');
-const ContentText = classed('p', 'mt-1 typo-callout text-text-tertiary');
+export const ContentHeading = classed('h2', 'font-bold typo-body');
+export const ContentText = classed('p', 'mt-1 typo-callout text-text-tertiary');
```

Comment:
Probably need to start exposing these to commons somewhere, but not blocking.

Reply:
`Typgraphy` not fixing this?

Reply:
I'll leave it out for now, and we could probably do a cleanup later if we move more things to `Typography`?

Reply:
Yeah can be later stage indeed :)